src/components/Header/Header.tsx

```diff
@@ -54,6 +55,7 @@ const Header = ({
     <div
       className={classNames(
         styles.header,
+        navbarStatus ? styles.positionFixed : "",
```

Comment:
conditional classNames should be formatted as

```
classNames(
        styles.header,
        { 
          [styles.positionFixed]: !!navbarStatus
          "some class": true/false,
          etc
        },
        ...
```

see [docs example](https://www.npmjs.com/package/classnames)